pkgs/by-name/al/alpaca-proxy/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "alpaca-proxy";
+  version = "2.0.9";
+
+  src = fetchFromGitHub {
+    owner = "samuong";
+    repo = "alpaca";
+    rev = "v${version}";
+    hash = "sha256-Rf8//4FeruVZZ//uba80z20XGUxycwF91Aa09fosRXI=";
+  };
+
+  vendorHash = "sha256-JEiHgyPJvWmtPf8R4aX/qlevfZRdKajre324UsgRm5Y=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.BuildVersion=v${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/alpaca $out/bin/alpaca-proxy
```

A short comment explaining why this is renamed would be helpful…
```suggestion
    # executable is renamed to alpaca-proxy, to avoid collision with the alpaca python application
    mv $out/bin/alpaca $out/bin/alpaca-proxy
```

---

That was a good point. Applied :D